### PR TITLE
PRO-3083 & PRO-3102: Deprecate use of Bio Auth Service /templatizer & update comments

### PR DESCRIPTION
**Pull request by jeffk-kiva**
Summary of Changes
---
* Calls to Bio Auth Service should use /save instead of /templatizer.
* Update comments throughout the repo to reference Bio Auth Service instead of Identity Service.

Note: Protocol integration tests won't pass until this PR is merged and an image is pushed: https://github.com/kiva/guardian-bio-auth/pull/29

Signed-off-by: Jeff Kennedy <jeffk@kiva.org>

### CODE REVIEW

File: src/remote/dto/bio.auth.save.params.dto.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { FingerprintTypeEnum } from '../fingerprint.type.enum';
+import { IsEnum, IsInt, IsISO8601, IsNumber, IsOptional, IsString, Length } from 'class-validator';
+
+/**
+ * DTO for the params of each fingerprint in the body of the request sent to Bio Auth Service's /save endpoint
+ */
+export class BioAuthSaveParamsDto {
+    @IsInt() readonly type_id: number;
```
Comment (jsaur) [inline src/remote/dto/bio.auth.save.params.dto.ts:8]: What about position?

**Review by jsaur — COMMENTED** [review]
Overall looks good - but may be missing position on BioAuthSaveParamsDto ?

File: src/remote/dto/bio.auth.save.params.dto.ts (hunk 1 above)
Comment (jeffk-kiva) [reply src/remote/dto/bio.auth.save.params.dto.ts:8]: Ack, thank you, good catch.

Comment (jeffk-kiva) [reply src/remote/dto/bio.auth.save.params.dto.ts:9]: Fixed in the latest and tested against protocol's integration tests.

**Review by jsaur — APPROVED** [review]
👍

**Pull request merged**